Content.Shared/CharacterAppearance/Systems/SharedHumanoidAppearanceSystem.cs

```diff
@@ -69,6 +74,28 @@ public void UpdateSexGender(EntityUid uid, Sex sex, Gender gender, HumanoidAppea
             RaiseLocalEvent(uid, new ChangedHumanoidAppearanceEvent(component.Appearance, component.Sex, component.Gender, component.Species), true);
         }
 
+        /// <summary>
+        /// Takes ID of the species prototype, returns UI-friendly name of the species.
+        /// </summary>
+        public string GetSpeciesRepresentation(string speciesId) {
+            var prototypeManager = IoCManager.Resolve<IPrototypeManager>();
```

Comment:
you dont need to manually resolve, store it as a `[Dependency]` in a system field